src/api/requirements.txt

```diff
@@ -216,5 +217,6 @@ zipp==3.4.1
 dacite==1.6.0
 PyYAML==6.0
     # added manually by denny. these might get overwritten
+ultralytics==8.0.145
```

Comment:
aatman, can you review this section? we should not be adding operator specific requirements to the requirements txt file. we want to keep the feluda core to be very lean. look at other operators and how they specify their requirements uptop like this - https://github.com/tattle-made/feluda/blob/79d67ee01653aa52dc2aba38efc34ad4f2f5b1de/src/api/core/operators/image_phash.py#L6

Reply:
some errors were coming when I did that because ultralytics depends on many other libraries like `opencv`, `Pillow`, `tqdm` etc. But let me try installing it in the operator and update the code